examples/fonts-custom/docs/README.md

```diff
@@ -0,0 +1,145 @@
+for your own site, without assuming what fonts you might want to use.
+We also describe how to use custom fonts that are hosted on your own
+server, see below.
+
+Here we have some text in:
+
+- Noto Serif
+- **Noto Serif bold**
+- *Noto Serif italic*
+- __*Noto Serif bold italic*__
+- `Noto Mono`
+- __`Noto Mono`__ 
+
+## Hot it works
+
+In `mkdocs.yml`, turn off the auto-loading of fonts from Google Fonts
+and add an extra CSS stylesheet to configure your custom fonts.
+
+```yaml
+theme:
+  font: false
+
+extra_css:
+  - assets/stylesheets/extra.css
+```
+
+Then, [configure custom fonts] hosted either on a CDN or on your own
+server, depending on your needs. We demonstrate this here with the 
+Noto Sans and Noto Mono font families.
+
+[configure custom fonts]: https://squidfunk.github.io/mkdocs-material/setup/changing-the-fonts/#additional-fonts
+
+### Using a CDN's CSS
+
+If you are using a CDN to load your fonts, you may want to use CSS
+that the operators of the CDN are providing for defining the font
+faces, instead of writing your own. You still need to configure the
+fonts for Material to use, though:
+
+=== "`mkdocs.yml`"
+
+    ```yaml
+    extra_css:
+      - https://fonts.cdnfonts.com/css/noto-serif
+      - https://fonts.cdnfonts.com/css/noto-mono
+      - assets/stylesheets/extra.css
+    ```
+
+=== "`extra.css`"
+
+    ```css
+    :root {
+      --md-text-font: "Noto Serif";
+      --md-code-font: "Noto Mono";
+    }
+    ```
+
+### Using a CDN with your own CSS
+
+=== "`mkdocs.yml`"
+
+    ```yaml
+    extra_css:
+      - assets/stylesheets/extra.css
+    ```
+
+=== "`extra.css`"
+
+    ```css
+    @font-face {
+      font-family: "Noto Serif"; font-weight: normal; font-style: normal;
+      src: url("https://cdn.jsdelivr.net/gh/notofonts/notofonts.github.io/fonts/NotoSerif/unhinted/otf/NotoSerif-Regular.otf");
+    }
+
+    @font-face {
+      font-family: "Noto Serif"; font-weight: bold; font-style: normal;
+      src: url("https://cdn.jsdelivr.net/gh/notofonts/notofonts.github.io/fonts/NotoSerif/unhinted/otf/NotoSerif-Bold.otf");
+
+    }
+
+    @font-face {
+      font-family: "Noto Serif"; font-weight: normal; font-style: italic;
+      src: url("https://cdn.jsdelivr.net/gh/notofonts/notofonts.github.io/fonts/NotoSerif/unhinted/otf/NotoSerif-Italic.otf");
+
+    }
+
+    @font-face {
+      font-family: "Noto Mono"; font-weight: normal; font-style: normal;
+      src: url("https://cdn.jsdelivr.net/gh/notofonts/notofonts.github.io/fonts/NotoSansMono/unhinted/otf/NotoSansMono-Regular.otf");
+
+    }
+
+    @font-face {
+      font-family: "Noto Mono"; font-weight: bold; font-style: normal;
+      src: url("https://cdn.jsdelivr.net/gh/notofonts/notofonts.github.io/fonts/NotoSansMono/unhinted/otf/NotoSansMono-Bold.otf");
+    }
+
+    :root {
+      --md-text-font: "Noto Serif";
+      --md-code-font: "Noto Mono";
+    }
+    ```
+
+### Hosting on your own server
```

Comment:
I'd just call this "alongside your documentatin" or something like it. "Server" might be off-putting, because it sounds like you need to set up hosting yourself. Just put them in docs, deploy to GitHub pages, done.